### PR TITLE
XIVY-12086 add ivy project explorer

### PR DESCRIPTION
**Pull request by ivy-lmu**
image

### CODE REVIEW

File: vscode-extensions/project-explorer/extension/assets/ivy-logo.png
Comment (alexsuter) [inline vscode-extensions/project-explorer/extension/assets/ivy-logo.png:1]: svg?

File: vscode-extensions/project-explorer/extension/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "vscode-project-explorer-extension",
+  "private": true,
+  "version": "11.2.0",
+  "description": "VSCode extension that provides the Axon Ivy project explorer",
```
Comment (alexsuter) [inline vscode-extensions/project-explorer/extension/package.json:5]: The Axon Ivy Project Explorer is meticulously designed to seamlessly integrate Axon Ivy projects into your development workflow.

File: vscode-extensions/project-explorer/extension/package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "displayName": "Axon Ivy Project Explorer",
+  "publisher": "AxonIvy",
```
Comment (alexsuter) [inline vscode-extensions/project-explorer/extension/package.json:7]: Axon Ivy AG

Comment (ivy-lmu) [reply vscode-extensions/project-explorer/extension/package.json:7]: not possible:
image

File: vscode-extensions/project-explorer/extension/assets/ivy-logo.png
Comment (ivy-lmu) [reply vscode-extensions/project-explorer/extension/assets/ivy-logo.png:1]: https://code.visualstudio.com/updates/v1_14#_constraints-for-rendering-svg-images

File: vscode-extensions/project-explorer/extension/package.json (hunk 2 above)
Comment (alexsuter) [reply vscode-extensions/project-explorer/extension/package.json:7]: what are the restrictions?

Comment (ivy-lmu) [reply vscode-extensions/project-explorer/extension/package.json:7]: Expected the identifier of a publisher, not its human-friendly name.  Learn more: https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions

File: vscode-extensions/project-explorer/extension/package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./lib/extension.js",
+  "devDependencies": {
+    "@types/vscode": "^1.54.0",
+    "ts-loader": "^9.4.2"
+  },
+  "scripts": {
+    "build": "tsc -p ./",
```
Comment (ivy-cst) [inline vscode-extensions/project-explorer/extension/package.json:37]: I think it must be changed soon to a bundler, if we have common stuff on other packages ...

**Review by ivy-cst — APPROVED** [review]
Cool, easy :fish: 
Provider looks like a lot o low level code, but seems to be straight forward :+1: !

**Comment by ivy-cst** [comment]
A special Icon for the ivy project would be cool. May be also possible to show that in the original tree?

File: vscode-extensions/project-explorer/extension/package.json (hunk 3 above)
Comment (ivy-lmu) [reply vscode-extensions/project-explorer/extension/package.json:37]: agree, but the switch is not a big deal

**Comment by ivy-lmu** [comment]
> A special Icon for the ivy project would be cool. May be also possible to show that in the original tree?

added to subtasks 👍

**Review by ivy-cst — APPROVED** [review]
Nice :+1:

**Pull request merged**